internal/scaffold/helm/watches.go

```diff
@@ -15,8 +15,17 @@
 package helm
 
 import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
 	"github.com/operator-framework/operator-sdk/internal/scaffold"
 	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
+	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
+	"github.com/operator-framework/operator-sdk/pkg/helm/watches"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 )
```

Comment:
fix the order the imports. 

```
"fmt"
"io/ioutil"
"path/filepath"
"gopkg.in/yaml.v2"

log "github.com/sirupsen/logrus"
"k8s.io/apimachinery/pkg/runtime/schema"

	"github.com/operator-framework/operator-sdk/internal/scaffold"
	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
	"github.com/operator-framework/operator-sdk/pkg/helm/watches"
```

Reply:
@camilamacedo86 I don't understand the order you want. When I resave this file goimports is sorting them the same way and it passes the linter's gofmt. So I would NOT reorder these.